index.html

```diff
@@ -176,6 +176,19 @@ <h3 class="lang-font">Skills</h3>
           </div>
         </div>
     </section>
+    <section class="contact-section">
+      <h2 class="con-title">Contact me</h2>
+      <p class="con-paragraph">If you have an application you are interested in developing, a feature that
+          you
+          need built or a project that needs coding. I’d love to help with it</p>
+      <form action="https://formspree.io/f/mnqlovzq" method="POST" class="con-form">
+          <input type="text" name="_name" placeholder="Please enter your full name" class="form-text" required>
```

Comment:
- [ ]  Name input (can be first name and last name field depending on the template used) has `maxlength="30"` attribute.

Reply:
Done